frontend/app/sysupport-admin/index/controller.js

```diff
@@ -0,0 +1,66 @@
+import Controller from '@ember/controller'
+
+export default Controller.extend({
+  init() {
+    this._super(...arguments)
+    this.set('headers', [
```

Comment:
This would be a use case for a computed property just returning this array

Reply:
:+1: